code/modules/mob/mob.dm

```diff
@@ -492,7 +492,7 @@
 		return FALSE
 
 	//you can only queue up one examine on something at a time
-	if(examined_thing in do_afters)
+	if(DOING_INTERACTION(src, DOAFTER_SOURCE_FEELYTOUCHY))
```

Comment:
I believe this can just be
`DOING_INTERACTION_WITH_TARGET(src, examined_thing)`
instead of needing its own interaction key, because they're supposed to be able to examine multiple atoms at once, but they're NOT supposed to be able to examine the same atom at the same time (I tested it out in the test merge and it didn't operate like it should)

Reply:
good point thank u